Application/Client Side/client.py

```diff
@@ -15,111 +15,114 @@
+        The file will save in the same path of client.py
+        """
+        try:
+            filename = input("Enter file name: ")
+            file_content = input("Enter file content: ")
+            encrypt = input("Encrypt file content? (y/n): ")
+            if encrypt == 'y':
+                encrypt_key = input("Enter encrypt key: ")
+                # Create Fernet object with key
+                cipher_suite = Fernet(encrypt_key)
+                # Encrypt file content
+                file_content = cipher_suite.encrypt(
+                    file_content.encode('utf-8')).decode('utf-8')
+            with open(os.path.join(sys.path[0], filename), 'w') as f:
+                f.write(str(file_content))
+            return (True, file_content)
+        except IOError as e:
+            print(f"Failed to create file: {e}")
+            return False
 
 
 def main():
-    """
-    Drive the main function to run the function of client side.
-    Support user to choose to send dictionary or file.
-    """
-    try:
-        while True:
-            item_type = input(
-                "Enter '1' to send a dictionary or '2' to send a file: ")
-            if item_type == '1':
-                my_dict = {}
-                while True:
-                    key = input("Enter a key or 'q' to quit: ")
-                    if key == 'q':
-                        break
-                    value = input("Enter a value: ")
-                    my_dict[key] = value
-                serialization_format = input(
-                    "Enter serialization format (binary, JSON or XML): ")
-                send_dictionary(my_dict, serialization_format, HOST, PORT)
-            elif item_type == '2':
-                my_file = create_file()
-                send_file(my_file, HOST, PORT)
-            else:
-                print("Invalid input. Please enter '1' or '2'.")
-    except Exception as e:
-        print(f"An error occurred in the main function: {e}")
+        """
+        Drive the main function to run the function of client side.
+        Support user to choose to send dictionary or file.
+        """
+        try:
+            while True:
+                item_type = input(
+                    "Enter '1' to send a dictionary or '2' to send a file: ")
+                if item_type == '1':
+                    my_dict = {}
+                    while True:
+                        key = input("Enter a key or 'q' to quit: ")
+                        if key == 'q':
+                            break
+                        value = input("Enter a value: ")
+                        my_dict[key] = value
+                    serialization_format = input(
+                        "Enter serialization format (binary, JSON or XML): ")
+                    send_dictionary(my_dict, serialization_format, HOST, PORT)
+                elif item_type == '2':
+                    my_file = create_file()
+                    send_file(my_file, HOST, PORT)
```

Comment:
the function of send_file() is getting the content returned by create_file(). So the content can't send to server-side if the return of create_file() is bool.